### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -0,0 +1,2025 @@
...
+    "name": "brazeIdentifier",
+    "displayName": "Identity settings",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "TEXT",
+        "name": "brazeExternalUserId",
+        "displayName": "Braze external_id",
+        "simpleValueType": true,
+        "help": "Specify the client event property to use as the `external_id` user identifier for Braze API.",
```
Comment (paulboocock) [inline template.tpl]: Perhaps worth adding to the help what a recommended value is? I assume `client_id` would usually be what people would use?

File: template.tpl [hunk 2]
```diff
@@ -0,0 +1,2025 @@
...
+  const isoTime = [pad(hrs, 2), pad(mins, 2), pad(secs, 2)].join(':');
+
+  return isoDate + 'T' + isoTime + '.' + pad(msecs, 3) + 'Z';
+}
+
+/*
+ * Transforms a unix timestamp in milliseconds to ISO-8601 time code.
+ * Ref: https://howardhinnant.github.io/date_algorithms.html#civil_from_days
+ */
+function unixMillisToISO(unixMillis) {
```
Comment (paulboocock) [inline template.tpl]: This is fun 😬 

Looks good though!

**Review by paulboocock — APPROVED** [review]
LGTM. One very minor nit.

File: template.tpl (hunk 1 above)
Comment (adatzer) [reply template.tpl]: Thank you for reviewing @paulboocock !
Added the client_id as an "e.g." in the help text.

File: template.tpl (hunk 2 above)
Comment (adatzer) [reply template.tpl]: Indeed, it was standard-JavaScript fun! Unit tested it as much as i could and seems consistent so far..

File: metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+homepage: "https://www.snowplowanalytics.com"
+documentation: "https://docs.snowplowanalytics.com/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/"
```
Comment (paulboocock) [inline metadata.yaml]: Might as well publish the docs and update this link before we submit it to Google.

**Pull request merged**